.env

```diff
@@ -0,0 +1 @@
+API_URL=http://localhost:3000/api/v1
```

Comment:
Great work Halima. Is there any reason why you're using localhost for this variable

Reply:
I used localhost to test the changes I made in the backend before it got merged and deployed and I forgot to remove it. Thanks for noting that.